### PR TITLE
Fei Yan's datasets

### PR DESCRIPTION
**Pull request by kernelogic**

### CODE REVIEW

File: recovery/cids/restore/huge-stock-market-dataset.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (RobQuistNL) [inline recovery/cids/restore/huge-stock-market-dataset.csv:1]: superfluous space here

**Review by RobQuistNL — CHANGES_REQUESTED** [review]

**Comment by orvn** [comment]
@kernelogic, did you mean to entirely remove your CIDs from `genome-in-a-bottle` and `huge-stock-market-dataset` (where there's one left)?

**Comment by kernelogic** [comment]
@orvn genome-in-a-bottle.csv is just a placeholder, I haven't fully packed the dataset as cars yet.
huge-stock-market-dataset.csv is very small, containing only one car, it is expected.

**Review by orvn — APPROVED** [review]
That makes sense @kernelogic, merging

**Pull request merged**